### ISSUE TITLE
1.20.1 Gregtech crashing

### ISSUE THREAD

**Issue by funnycubeguy** [IzzelAliz/Arclight#1349]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

1.20.1-1.0.6-SNAPSHOT-52ab6e5

### OS & Java versions

Debian 12 Java 17 Pterodactyl

### Plugins and Mods

```raw
https://pastes.dev/eKGrBeiK2v


https://www.curseforge.com/minecraft/mc-mods/gregtechceu-modern/files/5311877
```

### Description

Crashes when you open and close a gregtech super chest.

Tried various archlight versions (the 1.0.5 too) with no luck.

### Server pack link (Optional)

ATM9

**Comment by funnycubeguy** [comment]
Issue is for all gregtech machines on all version of arclight!

**Comment by RogueLightt** [comment]
I am running a server also installed with arclight and gregtech and seem to having the same issue, everything installed correctly and updated but still seems to be encountering crashes upon interaction with gregtech blocks

**Comment by KAPAMEJlbKA** [comment]
I have exactly the same problem

**Comment by Ap4uuk** [comment]
++1 i have a too problem this mod
https://pastes.dev/1uns986fp4

**Comment by KAPAMEJlbKA** [comment]
I fixed it.

**Comment by RogueLightt** [comment]
> I fixed it.

How did you fix?

**Comment by funnycubeguy** [comment]
> I fixed it.

whats the fix?

**Comment by KAPAMEJlbKA** [comment]
I changed the kernel code in the place where we get the GUI title.  Added processing of key to NULL

**Comment by RogueLightt** [comment]
> I changed the kernel code in the place where we get the GUI title. Added processing of key to NULL

Do you have a discord by chance? or can you provide a bit more in depth explanation? 

YOU ARE AMAZING

**Comment by KAPAMEJlbKA** [comment]
Yes, I have a discord. I don't speak English well, but I can give explanations. Here is my discord KAPAMEJlbKA#7274
